### ISSUE TITLE
Bitdefender Flagged and Removed the .exe

### ISSUE THREAD

**Issue by deeseem** [IgorMundstein/WinMemoryCleaner#48]
### Checklist

- [X] I have used the search function to see if someone else has submitted the same bug report.
- [X] I will describe the problem in as much detail as possible.

### App version

2.5

### Windows version

Win 10

### Steps To Reproduce

Have Bitdefender installed. Both programs running together fine with no flags or issues. Run Bitdefender after latest update. Get a "critical" warning that says Bitdefender has removed Winmemcleaner due to suspicious behaviour.

### Current Behavior

I thought it was unlikely to be a legitmate move by Bitdefender, so restored the program.  Screenshot of Bitdefender explanation here: https://ibb.co/kqxTtqX

### Expected Behavior

Not sure I added it to exceptions when I restored it, but presume it would have happened automatically when I did.

### Anything else?

![BB winmemclr](https://github.com/IgorMundstein/WinMemoryCleaner/assets/129421414/f094fc5e-88b0-4e0e-891a-a5e01bc0f961)

https://ibb.co/kqxTtqX

![BB winmemclr](https://github.com/IgorMundstein/WinMemoryCleaner/assets/129421414/f47ecdbf-33ed-4015-80a5-ad09f2cc5e78)

**Comment by IgorMundstein** [comment]
Thank you for taking the time to make this report.

As I often do with Microsoft. I submitted to Bitdefender a false positive report about this. I hope they fix it soon.

[Incorrect Detection: Report a False Positive or False Negative detection to Bitdefender Labs](https://www.bitdefender.com/consumer/support/answer/29358/)

![image](https://github.com/IgorMundstein/WinMemoryCleaner/assets/3512138/62d58aab-7010-41fc-851b-5b88f0ca78c8)

**Comment by deeseem** [comment]
Hi Igor,

Just to let you know, I restored it from Bitdefenders vault or whatever
they call it, then ran a scan on it with Bitdefender and it came up clean.
Norton found absolutely nothing wrong with it at all either, unsurprisingly
and didn't flag it on my other machine.

I don't know if restoring it made it now an accepted file to Bitdefender
tho I expect exactly so, but the screenshot I attached does clearly show
that after weeks of both running together on the same machine fine, it was
suddenly quarantined/removed.

It's a shame, as a lot of people would simply trust an anti-virus program,
95% of most of which function primarily to try and convince the user/buyer
that their purchase is actually doing something worthwhile, and that sites
that haven't renewed their security certificates are suddenly unsafe when
nothing has typically changed other than they didn't pay someone to stop
anti-viruses blocking traffic.
